proto/v1beta1/common.proto

```diff
@@ -39,6 +39,8 @@ enum NoteKind {
   DISCOVERY = 6;
   // This represents a logical "role" that can attest to artifacts.
   ATTESTATION = 7;
+  // This represents an in-toto link file.
```

Comment:
This comment is a bit too specific in in-toto terminology. I recommend to expand on what "in-toto link file" means in terms of software supply chain metadata.